app/services/email_digest_article_collector.rb

```diff
@@ -49,20 +49,7 @@ def days_until_next_email
     # Relies on hyperbolic tangent function to model the frequency of the digest email
     max_day = SiteConfig.periodic_email_digest_max
     min_day = SiteConfig.periodic_email_digest_min
-    result = max_day * (1 - Math.tanh(2 * open_rate))
-    result = result.round
-
-    [result, min_day].max
-  end
-
-  def open_rate
-    email_count = last_user_emails.count
-
-    # Will stick with 50% open rate if @user has no/not-enough email digest history
-    return 0.5 if email_count < 10
-
-    past_opened_emails_count = last_user_emails.count { |msg| msg.opened_at.present? }
-    past_opened_emails_count / email_count
+    rand(min_day..max_day)
```

Comment:
Since we can no longer use any calculations based on the open rate to determine when to send the next email digest, we'll just pick a random number between the min and max days specified in config.

Reply:
Great point! Any thoughts on this @forem/support ?

Reply:
I'm not @michael-tharrington but it looks to me that relinquishing this data tracking (which I legit despise on a purely personal level) is worth the fix. But as I say, I'm biased and don't speak for the whole of Community Support.

And I'm also not Marketing so don't know how dependent we are on that data, besides timing email frequency.

Reply:
I'd agree with @ellativity that relinquishing the data tracking sounds worth the fix!

As far as Dev Digest emails go, everything I've heard is that folks receive them much more frequently than they'd expect. (see - https://github.com/forem/forem/issues/10940) So as long as this doesn't kick up the frequency of these emails then I think we're good.